pkg/combustion/templates/k8s-vip.yaml.tpl

```diff
@@ -27,7 +27,7 @@ metadata:
   name: endpoint-copier-operator
   namespace: endpoint-copier-operator
 spec:
-  repo: https://suse-edge.github.io/endpoint-copier-operator
+  repo: https://suse-edge.github.io/charts
```

Comment:
This is interesting when we consider production v. dev. Last I heard (and I may be wrong), our charts will be pulled from *BS. But I also see the need to be able to pull from this sort of development branch too. 

You might already have looked into this and it's a non-issue. But if we're going to have a different location for our shipped charts, can you create an issue so we remember to go back and update this to pull from there?